test/e2e/scheduling/preemption.go

```diff
@@ -376,7 +376,7 @@ var _ = SIGDescribe("SchedulerPreemption [Serial]", func() {
 			},
 			Affinity: &testNodeAffinity,
 		})
-		framework.Logf("Created pod: %v", preemptorPod.Name)
+		framework.Logf("Created pod: %v on node: %s", preemptorPod.Name, preemptorPod.Spec.NodeName)
```

Comment:
am I miss something, this looks like to me just some log, how does this fix the flaky?

Reply:
There is no fix yet, marked the PR as WIP and added a comment to the reviewer that it is used to reproduce the issue.